webapp/pages/terms-and-conditions.vue

```diff
@@ -1,14 +1,72 @@
 <template>
   <div>
     <ds-space>
-      <ds-heading tag="h2">{{ $t('site.termsAc') }}</ds-heading>
+      <ds-heading tag="h2">{{ $t('site.termsAc') }} ( vom: {{ latestDate_TaC }})</ds-heading>
     </ds-space>
+    <div v-if="confirmed === true">
+      <ds-section class="hasConfirm" primary>
+        <ds-flex>
+          <ds-flex-item>
+            <ds-logo inverse></ds-logo>
+          </ds-flex-item>
+          <ds-flex-item width="4">
+            <ds-heading>you have confirmed</ds-heading>
+            <ds-text>
+              You have confirmed the current Terms and Conditions on
+              {{ latestDate_TaC_UserConfirmed }}.
+              <br />
+              You will be informed if there is a change in the Terms and Conditions.
+            </ds-text>
+          </ds-flex-item>
+        </ds-flex>
+      </ds-section>
+    </div>
+    <div v-else>
+      <ds-section class="hasNoConfirm" secondary>
+        <ds-flex>
+          <ds-flex-item>
+            <ds-logo inverse></ds-logo>
+          </ds-flex-item>
+          <ds-flex-item width="4">
+            <ds-heading>Our terms and conditions have changed.</ds-heading>
+            <ds-text>
+              Please confirm the AGB. The AGB have changed on {{ latestDate_TaC_update }}!
+              <br />
+              Your last confirmation was on {{ latestDate_TaC_UserConfirmed }}.
+            </ds-text>
+          </ds-flex-item>
+          <ds-flex-item width="1">
+            <ds-button primary>confirm</ds-button>
+          </ds-flex-item>
+        </ds-flex>
+      </ds-section>
+    </div>
     <ds-container>
       <ds-space margin-top="large">
         <ds-text>{{ $t('site.termsAc') }}</ds-text>
         <ds-text v-html="$t('terms.text')"></ds-text>
       </ds-space>
     </ds-container>
+    <div v-if="confirmed === false">
+      <ds-section class="hasNoConfirm" secondary>
+        <ds-flex>
+          <ds-flex-item>
+            <ds-logo inverse></ds-logo>
+          </ds-flex-item>
+          <ds-flex-item width="4">
+            <ds-heading>Our terms and conditions have changed.</ds-heading>
+            <ds-text>
+              Please confirm the AGB. The AGB have changed on {{ latestDate_TaC_update }}!
```

Comment:
`AGB` is a German for "Allgemeine Geschäftsbedingungen". Obviously this cannot be translated to English.